cisco_aci/datadog_checks/cisco_aci/models.py

```diff
@@ -59,7 +59,16 @@ class DeviceMetadata(BaseModel):
         @computed_field
         @property
         def status(self) -> int:
-            return 1 if self.ad_st == 'on' else 2
+            mapping = {
+                'active': 1,
+                'inactive': 2,
+                'disabled': 5,
+                'discovering': 2,
+                'undiscovered': 2,
+                'unsupported': 2,
+                'unknown': 4,
+            }
+            return mapping.get(self.fabric_st, 7)
```

Comment:
What do the value 1-7 map mean?

Reply:
i was trying to find where i managed to get this mapping from and i have no idea anymore 😭 i checked the meraki crawler and it seems to be either 1/2 so i'm updating this!